generation/internal/jobs.go

```diff
@@ -60,7 +83,11 @@ func addStubJobs(jobs []*Job) []*Job {
 		jobs = append(jobs, &stubTestJob)
 	}
 	if !jobTypesPresent[DeployJob] {
-		jobs = append(jobs, &stubDeployJob)
+		deployJob := stubDeployJob
```

Comment:
Tests were failing if one of them altered the stubDeployJob, so I ended up making a copy of it before mutation to fix that.